src/theory/bags/inference_generator.cpp

```diff
@@ -261,9 +262,10 @@ Node InferenceGenerator::getMultiplicitySkolem(Node element,
                                                InferInfo& inferInfo)
 {
   Node count = d_nm->mkNode(kind::BAG_COUNT, element, bag);
-  Node skolem = d_state->registerBagElement(count);
+  Node skolem = d_state->registerCountTerm(count);
   eq::EqualityEngine* ee = d_state->getEqualityEngine();
   ee->assertEquality(skolem.eqNode(count), true, d_nm->mkConst(true));
+  // inferInfo.d_conclusions.push_back(skolem.eqNode(count));
```

Comment:
This commented line fails regressions whereas line 266 does not. This needs discussion.

Reply:
You need to be sure you are processing pending assertions.  I would remove the use of `ee` here for now.

Reply:
Done. It was a stupid mistake inside InferInfo::process